app/services/live.js

```diff
@@ -297,6 +300,13 @@ export default class LiveService extends Service {
 
   async renderScreenVideoToPeers(peers) {
     this.peers = peers;
+    const maven = peers.find((peer) => peer.roleName === ROLES.maven);
+    const moderator = peers.find((peer) => peer.roleName === ROLES.moderator);
+    const guest = peers.find((peer) => peer.roleName === ROLES.guest);
```

Comment:
we should optimize this, please think about how we can do this in one loop instead of 3 and make it scalable if  we add more roles

Reply:
i and Satyam tried doing but this was not working we had do like this

Reply:
if you have solution then please let me know

Reply:
done